td-logger/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+#![no_std]
+
+use log::{Level, Metadata, Record};
+use log::{LevelFilter, SetLoggerError};
+
+mod logger;
+pub use logger::*;
+
+macro_rules! tdlog {
+    ($($arg:tt)*) => (crate::logger::_log_ex(crate::logger::LOG_LEVEL_VERBOSE, crate::logger::LOG_MASK_COMMON, format_args!($($arg)*)));
```

Comment:
Question: Just checking that the first two params are as expected. It's fairly conventional to log at info level by default but we're logging at verbose level. So should verbose => info, or should `LOG_MASK_COMMON` => `LOG_MASK_ALL`?